### ISSUE TITLE
Mask Loss

### ISSUE THREAD

**Issue by brianneoberson** [turandai/gaussian_surfels#28]
Hello, 

I have a question about the mask loss. In the paper you mention that the mask loss is computed as the binary cross-entropy between $\sum_{i=0} T_i  \alpha_i$ and the mask. But in the code the mask loss is computed as such:

https://github.com/turandai/gaussian_surfels/blob/c1cbb40b6f048e7a13535c8e0fb210c8a8192966/train.py#L116

Is there a reason you changed it? Or is this somehow equivalent?

Thanks for your help!

**Comment by turandai** [comment]
Hi, both of the two loss push the ray transmittance to match the mask, we use this one as it is simpler. The ```opac``` used here is $1-\sum{T_i{\alpha}_i}$, and we use max polling to dilate the gt mask a little to prevent masking out the foreground object due to noise in gt mask.

**Comment by turandai** [comment]
Hi, both of the two loss push the ray transmittance to match the mask, we use this one as it is simpler. The ```opac``` used here is $\sum{T_i{\alpha}_i}$, and we use max polling to dilate the mask a little to prevent masking out the foreground object due to noise in mask.
